docs/source/api_commands.rst

```diff
@@ -15,10 +15,37 @@ Parameters: None
 app_get_dryer_setting
 ---------------------
 
-Description:
+Description: Get docker dryer settings.
 
-Parameters:
+Parameters: None
+
+Returns:
+
+    status:
+
+    on:
+
+        cliff_on:
+
+        cliff_off
+
+        count:
+
+        dry_time: Duration dryer remains on in seconds.
 
+    off:
+    
+        cliff_on:
+
+        cliff_off:
+
+        count:
+
+Return example::
+
+    {'status': 1, 'on': {'cliff_on': 1, 'cliff_off': 1, 'count': 10, 'dry_time': 7200}, 'off': {'cliff_on': 2, 'cliff_off': 1, 'count': 10}}
+
+Source: Roborock S7 MaxV Ultra
```

Comment:
Rather than a source, I think a supported would be helpful. A generic table for each one where each vacuum states if it is supported/ not supported/ unknown. And then if they ever return different results, we could add that when it occurs rather than always having, just my opinion